src/vcpkg/commands.ci.cpp

```diff
@@ -404,16 +404,14 @@ namespace vcpkg::Commands::CI
         auto& var_provider = *var_provider_storage;
 
         const ElapsedTimer timer;
-        std::vector<std::string> all_port_names =
-            Util::fmap(provider.load_all_control_files(), Paragraphs::get_name_of_control_file);
         // Install the default features for every package
-        std::vector<FullPackageSpec> all_default_full_specs;
-        all_default_full_specs.reserve(all_port_names.size());
-        for (auto&& port_name : all_port_names)
-        {
-            all_default_full_specs.emplace_back(PackageSpec{std::move(port_name), target_triplet},
-                                                InternalFeatureSet{"core", "default"});
-        }
+        const std::vector<FullPackageSpec> all_default_full_specs =
+            Util::fmap(provider.load_all_control_files(), [target_triplet](auto* scfl) {
```

Comment:
I think this is worse.

* It makes N extra copies because it constructs a `FullPackageSpec` temporary that is moved into the vector
* it's harder to debug because it
  * introduces templates that cause breakpoints to not stick
  * is lambda tastic so step over doesn't work

I think `Util::fmap` can increase clarity in some circumstances and in those places I think those costs are worth it but that's not true here.

Reply:
Sorry, perhaps I'm blind: Where is the vector<string> itermediate?

Reply:
```
std::vector<std::string> all_port_names =
            Util::fmap(provider.load_all_control_files(), Paragraphs::get_name_of_control_file);
```
was removed as part of this PR.